sfa_dash/templates/forms/base/base_report_form.html

```diff
@@ -18,6 +18,7 @@
 <form action="{{ url_for('forms.create_'+report_type+'_report') }}" method="post" id="report-form" onSubmit="return report_utils.validateReport();">
   <div class="form-group">
      <div class="form-element full-width">
+       <label for="start">Name:</label><br/>
```

Comment:
shouldn't the label be for "name"? Although now I see no ids for the inputs, so maybe these need a little more cleanup

Reply:
This is going to be kind of a big job to clean up, with a lot of mislabeling happening in javascript. I've opened up #352 as future work.